src/config.py

```diff
@@ -83,6 +83,10 @@ def set_sun_time():
 config["codeLightTheme"] = "Default Light+"
 config["codeDarkTheme"] = "Default Dark+"
 
+config["colorsEnabled"] = False
+config["colorsLightTheme"] = ''
+config["colorsDarkTheme"] = ''
+
```

Comment:
```suggestion
config["notificationsEnabled"] = True
```

this gives the flexibility to enable / disable the notifications